contracts/modules/grouping/GroupingModule.sol

```diff
@@ -209,6 +235,13 @@ contract GroupingModule is
         return pool.getAvailableReward(groupId, token, ipIds);
     }
 
+    /// @notice Checks whether a group reward pool is whitelisted
+    /// @param rewardPool The address of the group reward pool.
+    /// @return Whether the group reward pool is whitelisted.
+    function isWhitelistGroupRewardPool(address rewardPool) external view returns (bool) {
```

Comment:
This function may not be needed as people can call `isWhitelistedGroupRewardPool` in GROUP_IP_ASSET_REGISTRY. Or is it needed?

Reply:
agreed, remove this function from grouping module.